config/initializers/sidekiq.rb

```diff
@@ -1,3 +1,21 @@
+module Sidekiq
+  module Cron
+    class Job
+      def not_past_scheduled_time?(current_time)
+        last_cron_time = parsed_cron.previous_time(current_time).utc
+        # @mstruve/@sre: method monkey patched to increase time we look back for
```

Comment:
should we make this a configurable ENV variable which defaults to 60?

This way future installations not on Heroku can keep the default and/or play with it without having to commit a change

Reply:
Love that idea!